### PR TITLE
ignore exec tests when passing DOCKER_EXECDRIVER lxc

### PR DESCRIPTION
**Pull request by jessfraz**

### CODE REVIEW

File: project/make.sh [hunk 1]
```diff
@@ -95,6 +95,10 @@ if [ -z "$DOCKER_CLIENTONLY" ]; then
 	DOCKER_BUILDTAGS+=" daemon"
 fi
 
+if [ "$DOCKER_EXECDRIVER" == "lxc" ]; then
+	DOCKER_BUILDTAGS+=" test_no_exec"
```
Comment (tianon) [inline project/make.sh]: Single equals, and single quotes around `lxc` and ` test_no_exec`. :wink:

Comment (jfrazelle) [inline project/make.sh]: updated!

**Comment by jfrazelle** [comment]
@ashahab-altiscale this will make sure any tests around exec won't be run on lxc, get rid of a bunch of errors, let me know if you think I missed any

**Comment by ashahab-altiscale** [comment]
@jfrazelle This is awesome.
When I run my build, I see the following failures. Are these all accounted for?:

--- FAIL: TestExecApiCreateNoCmd (1.42s)
--- FAIL: TestExec (1.26s)
--- FAIL: TestExecInteractiveStdinClose (11.67s)
--- FAIL: TestExecInteractive (1.36s)
--- FAIL: TestExecAfterContainerRestart (2.51s)
--- FAIL: TestExecAfterDaemonRestart (4.49s)
--- FAIL: TestExecEnv (0.70s)
--- FAIL: TestExecExitStatus (1.30s)
--- FAIL: TestExecPausedContainer (2.29s)
--- FAIL: TestExecTtyCloseStdin (2.31s)
--- FAIL: TestExecTtyWithoutStdin (1.83s)

**Comment by jfrazelle** [comment]
Yep yep :)

**Comment by ashahab-altiscale** [comment]
In that case, LGTM.

**Comment by jfrazelle** [comment]
ping @tianon :angel:

**Comment by tianon** [comment]
LGTM

**Comment by ashahab-altiscale** [comment]
Can this me merged? @crosbymichael ? At some point I intend to implement exec, but right now it would be nice to ignore these tests for lxc.

**Comment by crosbymichael** [comment]
@jfrazelle needs a rebase

**Comment by jfrazelle** [comment]
rebased!

**Comment by ashahab-altiscale** [comment]
Is this test also an Exec test: TestRunMutableNetworkFiles? For me it fails on this line: https://github.com/docker/docker/blob/master/integration-cli/docker_cli_run_test.go#L2216

**Comment by jfrazelle** [comment]
op yep I missed one I can fix it, thanks for finding that :)

**Pull request merged**